js/npm/api-keys/fetchKeys.js

```diff
@@ -2,36 +2,34 @@ require("dotenv").config();
 const TheAuthAPI = require("theauthapi").default;
 
 const apiUrl = process.env.production
-    ? "https://api.theauthapi.com"
-    : process.env.TESTING_URL;
+  ? "https://api.theauthapi.com"
+  : process.env.TESTING_URL;
 const theAuthAPI = new TheAuthAPI(process.env.ACCESS_TOKEN, { host: apiUrl });
 
-
 async function fetchKeysSample() {
   // fetch all api keys
   const apiKeys = await theAuthAPI.apiKeys.getKeys();
-  console.log('all keys', apiKeys);
+  console.log("all keys", apiKeys);
 
-  // fetch keys with a specific name
+  // fetch keys with a specific name (1st one from the list returned previously)
   const nameFilteredKeys = await theAuthAPI.apiKeys.getKeys({
-    name: "new api key"
+    name: apiKeys[0].name,
   });
-  console.log('Keys filtered using name', nameFilteredKeys);
+  console.log("Keys filtered using name", nameFilteredKeys);
 
   // fetch keys where customUserId is null
   const customUserIdFilteredKey = await theAuthAPI.apiKeys.getKeys({
-    customUserId: null,
-  })
-  console.log('Keys filtered using customUserId', customUserIdFilteredKey);
+    customUserId: "null",
```

Comment:
I think leaving this as `null` instead of `"null"` is better, it feels more natural to pass `null` as `null` instead of a string